public/app/features/explore/state/query.ts

```diff
@@ -725,11 +725,14 @@ export const runLoadMoreLogsQueries = createAsyncThunk<void, RunLoadMoreLogsQuer
     let newQuerySource: Observable<ExplorePanelData>;
 
     const logQueries = queryResponse.logsResult?.queries || [];
-    const queries = logQueries.map((query: DataQuery) => ({
-      ...query,
-      datasource: query.datasource || datasourceInstance?.getRef(),
-      refId: `${infiniteScrollRefId}${query.refId}`,
-    }));
+    const logRefIds = queryResponse.logsFrames.map((frame) => frame.refId);
+    const queries = logQueries
```

Comment:
Just a small nit: wouldn't it make more sense to swap these two variable names? 
`queries`->`logQueries`; `logQueries` -> `queries`

Reply:
Absolutely, it looks better that way. Thanks!